### PR TITLE
WIP: Update README.md

### PR DESCRIPTION
**Pull request by raquelromano-gov**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -237,9 +238,9 @@ run on real browser, either because the tests is exercising browser quirks or be
 the test requries features that jsdom does not provide, putting them into a
 `e2e.spec.js` file is completely valid and good.
 
-TODO(awong): Figure out sauce labs integrations. Not all e2e tests should always be
-run on all browsers. That's wasteful. How do we determine what should be run on
-multiple browsers as opposed to on PhantomJS in Jenkins?
+### End-to-end Test -- Sauce Labs
+
+TODO: Translate internal documentation on running tests with Sauce Labs into something public.
```
Comment (raquelromano-gov) [inline README.md]: Should we just delete this section altogether since we can't share our Sauce Labs credentials publicly?

Comment (raquelromano-gov) [reply README.md]: This is our internal documentation: https://github.com/department-of-veterans-affairs/vets.gov-team/blob/master/Work%20Practices/Cross-Platform%20Testing/cross-platform-testing-plan.md

Comment (cvalarida) [reply README.md]: :thinking: Yeah, I can see it go either way, but I think it wouldn't be super useful without the credentials.

Comment (raquelromano-gov) [reply README.md]: OK, removed the section.

**Review by cvalarida — APPROVED** [review]

**Pull request closed without merging**